File: LivrusVirtus/DotNet/VCPP.NET/Livrus.cpp
#include "stdafx.h"
#include "Principal.h"

using namespace Livrus;

void __stdcall WinMain()
{
	System::Threading::Thread::CurrentThread->ApartmentState = System::Threading::ApartmentState::STA;

//	Application::Run(new frmPrincipal());


//	if (frmPrincipal::fPrincipal==null) {
		frmPrincipal::fPrincipal = new frmPrincipal();

		frmPrincipal::fPrincipal->db = new OleDbConnection();		
		frmPrincipal::fPrincipal->db->ConnectionString = 
                S"File Name=c:\\livrusvirtus\\bancos\\access\\access2k_livrus.udl";

		Application::Run(frmPrincipal::fPrincipal);
//	} else
//	  MessageBox::Show(this,S"a aplicação já está sendo executada.",
//	  S"livrus virtus", MessageBoxButtons::Ok);

}

File: LivrusVirtus/DotNet/JScript.NET/Livrus.js
import Livrus;

function main(): void {
  if (frmPrincipal.fPrincipal==null) {
     frmPrincipal.fPrincipal = new frmPrincipal();
    
     frmPrincipal.fPrincipal.db = new System.Data.OleDb.OleDbConnection();
     frmPrincipal.fPrincipal.db.ConnectionString=
     "File Name=C:\\LivrusVirtus\\bancos\\access\\Access2k_Livrus.udl";
    
      Application.Run(frmPrincipal.fPrincipal);

  } else
  MessageBox.Show("A Aplicação já está sendo executada.",
  "Livrus Virtus", MessageBoxButtons.OK);
}

main();